src_py/apiServer/globalVars.py

```diff
@@ -22,7 +22,8 @@
 E_TRAINING = 0
 E_PREDICTION = 1
 
-jsonPathLocation = 'NErlNet/jsonPath'
+username = os.getlogin()
+jsonPathLocation = '/home/{}/workspace/NErlNet/jsonPath'.format(username)
```

Comment:
This is not a good idea to point to Nerlnet under workspace. 
@dordor7  - Use the following: 
As part of the the new Nerlnet installer script there is a reserved symlink of Nerlnet (after calling NerlnetInstall) 
that always points to /usr/local/lib/nerlnet-lib 

https://github.com/leondavi/NErlNet/blob/master/NerlnetInstall.sh